Examples/Io/HepMC3/include/ActsExamples/Io/HepMC3/HepMC3Options.hpp

```diff
@@ -0,0 +1,44 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/Utilities/OptionsFwd.hpp"
+
+#include "HepMC3Reader.hpp"
+#include "HepMC3Writer.hpp"
+
+namespace ActsExamples {
+
+namespace Options {
+
+/// @brief HepMC3 specific writer options
```

Comment:
```suggestion
/// Add HepMC3 specific writer options.
```
Nitpicking: `@brief` is not required.

Reply:
done